packages/trader/src/Modules/Trading/Components/Form/Purchase/contract-info.tsx

```diff
@@ -72,7 +54,7 @@ const ContractInfo = ({
     };
 
     const has_error_or_not_loaded = proposal_info.has_error || !proposal_info.id;
-    const basis_text = has_error_or_not_loaded ? stakeOrPayout() : proposal_info.obj_contract_basis.text;
+    const basis_text = has_error_or_not_loaded ? stakeOrPayout() : proposal_info?.obj_contract_basis?.text || basis;
```

Comment:
@kate-deriv please don't add `|| basis` just to satisfy TS. please use `?? ''`. :)